src/arrayMethodSort.js

```diff
@@ -3,9 +3,46 @@
 /**
  * Implement method Sort
  */
+
+const compareStrings = (previous, current) => {
+  const previousString = previous.toString();
+  const currentString = current.toString();
+
+  if (previousString > currentString) {
+    return 1;
+  }
+
+  if (previousString === currentString) {
+    return 0;
+  }
+
+  return -1;
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareStrings) {
+    let sorted;
```

Comment:
it is better to always assign a value to a variable
`let sorted = false;`
so that the programmer working with this code in the future will understand what is in the variable from the start